### PR TITLE
using getAllServiceLocations in user_admin.php to retrieve facility list

### LINKED ISSUE #1541: user's default facility dropdown
after adding a second facility am unable to select it for the user's default facility

user = physician or clinician (might want to think adding a type of provider or practitioner for those sweet nurses)

### PR DESCRIPTION
**Pull request by odra**
Fixes #1541 

#### Short description of what this resolves:

* Lists a collection of facilities in the user "default facility" field as long as it is flagged as a "service location"

#### Changes proposed in this pull request:

- uses `getAllServiceLocations` instead of `getPrimaryBillingLocation` to load available facilities

### CODE REVIEW

File: interface/usergroup/user_admin.php [hunk 1]
```diff
@@ -315,7 +315,7 @@ function authorized_clicked() {
 <td><span class=text><?php echo xlt('Last Name'); ?>: </span></td><td><input type="text" name=lname id=lname style="width:150px;"  class="form-control" value="<?php echo attr($iter["lname"]); ?>"><span class="mandatory"></span></td>
 <td><span class=text><?php echo xlt('Default Facility'); ?>: </span></td><td><select name=facility_id style="width:150px;" class="form-control">
 <?php
-$fres = $facilityService->getAllBillingLocations();
+$fres = $facilityService->getAllServiceLocations();
```
Comment (odra) [inline interface/usergroup/user_admin.php:318]: I just switched the method here but I would like to know if we need a new method to list both billing and service locations or if using `getAllServiceLocations` is enough.

Comment (sjpadgett) [reply interface/usergroup/user_admin.php:318]: Good insight. I thought about this when I did fix and decided if it's only a billing location there'd be no reason for user to be defaulted to just a billing location.

Comment (stephenwaite) [reply interface/usergroup/user_admin.php:318]: good question and good point, think we're safe with `ServiceLocations` for now :)

**Comment by stephenwaite** [comment]
thanks for the contrib @odra

**Pull request merged**